### ISSUE TITLE
bootstrap 3

### ISSUE THREAD

**Issue by newtonianb** [wardrobecms/core-archived#79]
Is there a reason this has not been merged into master? I'm looking now its close to 29 commits behind?

**Comment by ericlbarnes** [comment]
Yes I started thinking a big framework like that would be over kill.  I'm going to stick with bootstrap v2 until we start work on wardrobe v2. I plan on changing it then.

**Comment by rtablada** [comment]
And with v2 you'll switch to some custom Gumby right?
